src/storm-dft/storage/elements/DFTGate.h

```diff
@@ -66,7 +66,7 @@ class DFTGate : public DFTChildren<ValueType> {
             queues.checkRestrictionLater(restr);
         }
         state.setFailed(this->mId);
-        this->childrenDontCare(state, queues);
+        this->childrenDontCare(queues);
```

Comment:
I guess this is what you want, but I was surprised that you can mark things not just in the state that you are considering? As in, i dont understand what is happening here.

Reply:
The line just adds the children to a queue for future consideration. Another function is then taking care of setting the state.
I agree that this is quite confusing and I think it can be improved. I would opt to make a separate PR for this though.

Reply:
Sure! It is actually better that it is now explicit that this code somehow does not require the state as an argument, but it does confuse me a lot :-)

Reply:
Issue https://github.com/moves-rwth/storm/issues/470 keeps track of this part.